mvp.feature

```diff
@@ -119,14 +119,14 @@ Feature: User with no Author identity in the Authority Register for Personas see
     And the user's project participation is known via their previously registered publications in NVA
     When they click the Projects search box
     And they see a drop-down containing ten active projects in Cristin Projects DB, ranked-by-year, filtered results
-        | their previously registered projects in NVA                                                           |
-        | any projects registered in the Cristin Projects DB with their ORCID                                   |
-        | any projects associated with a Grant ID from metadata from the Link lookup in the Cristin Projects DB |
-        | search results based by free-text search for the the user's name in the Cristin Projects DB           |
+      | their previously registered projects in NVA                                                           |
+      | any projects registered in the Cristin Projects DB with their ORCID                                   |
+      | any projects associated with a Grant ID from metadata from the Link lookup in the Cristin Projects DB |
+      | search results based by free-text search for the the user's name in the Cristin Projects DB           |
     And they select a Project
     Then the Project ID is added to the Publication metadata
     And the available data about people associated with the project is available to the NVA application
```

Comment:
Hva betyr det at `the available data about people associated with the project is available to the NVA application`? Skal disse personene legges inn som contributors? Skal de få en slags referanse til prosjektet i vår DB?